From 4f97631857831857204a7eb5a0e88d736a7db107 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Fri, 21 Apr 2017 11:16:27 +0100
Subject: [PATCH 1/4] Win32 + CONDA_BUILD: Do not use bundled ffmpeg

We use libvpx and libopus and not h264 in our own build of
ffmpeg and do not want to use the standard pre-built binary
which requires the user to download Cisco's OpenH64 if they
want h264 support.
---
 cmake/OpenCVFindLibsVideo.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/OpenCVFindLibsVideo.cmake b/cmake/OpenCVFindLibsVideo.cmake
index 1eff03b58..c7548803f 100644
--- a/cmake/OpenCVFindLibsVideo.cmake
+++ b/cmake/OpenCVFindLibsVideo.cmake
@@ -215,7 +215,7 @@ endif(WITH_XIMEA)
 # --- FFMPEG ---
 ocv_clear_vars(HAVE_FFMPEG)
 if(WITH_FFMPEG)
-  if(WIN32 AND NOT ARM)
+  if(WIN32 AND NOT (ARM OR DEFINED{CONDA_BUILD}))
     include("${OpenCV_SOURCE_DIR}/3rdparty/ffmpeg/ffmpeg.cmake")
     set(HAVE_FFMPEG TRUE)
   elseif(PKG_CONFIG_FOUND)
-- 
2.11.1

